src/lib/molecules/agenda-carousel.svelte

```diff
@@ -0,0 +1,164 @@
+<script>
+	import { AgendaCard } from '$lib/index.js';
+	import { onMount } from 'svelte';
+	let wrapper;
+
+	export let agendaItems = [
+		{
+			event: '{event}',
+			date: '{date}',
+			address: '{address}',
+			price: '{price}',
+			image: '{image}'
+		}
+	];
+
+	// onMount(async () => {
```

Comment:
Remove the comment if you're not using it.. production shouldn't have commented code..

Reply:
removed